debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java

```diff
@@ -568,6 +568,28 @@ public void parseGeomCollection() {
         assertThat(tables.forTable(new TableId(null, null, "geomtable"))).isNotNull();
     }
 
+    @Test
+    @FixFor("DBZ-1203")
+    public void parseAlterEnumColumnWithNewCharacterSet() {
+        String ddl =
+                "CREATE TABLE `test_stations_10` (\n" +
+                        "    `id` int(10) unsigned NOT NULL AUTO_INCREMENT,\n" +
+                        "    `name` varchar(500) COLLATE utf8_unicode_ci NOT NULL,\n" +
+                        "    `type` enum('station', 'post_office') COLLATE utf8_unicode_ci NOT NULL DEFAULT 'station',\n" +
+                        "    `created` datetime DEFAULT CURRENT_TIMESTAMP,\n" +
+                        "    `modified` datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+                        "    PRIMARY KEY (`id`)\n" +
+                        ");\n" +
+                        "\n" +
+                        "ALTER TABLE `test_stations_10`\n" +
+                        "    MODIFY COLUMN `type` ENUM('station', 'post_office', 'plane', 'ahihi_dongok', 'now')\n" +
+                        "    CHARACTER SET 'utf8' COLLATE 'utf8_unicode_ci' NOT NULL DEFAULT 'station';\n";
+
+        parser.parse(ddl, tables);
+        assertThat(tables.size()).isEqualTo(1);
+        assertThat(tables.forTable(new TableId(null, null, "test_stations_10"))).isNotNull();
```

Comment:
Perhaps add an assertion of the actual enum constants (if doable, I'm not sure how it is exposed, might be the "type expression" of the column)?

Reply:
@gunnarmorling  It is tested in `MySqlEnumColumnIT.java` - at the schema level

Reply:
Ah yes, thanks.